modules/ww-tritonnp/ww-tritonnp.wdl

```diff
@@ -0,0 +1,190 @@
+
+
+version 1.0
+
+# Import testdata module for automatic demo functionality
+import "https://raw.githubusercontent.com/caalo/wilds-wdl-library/refs/heads/ww-TritonNP/modules/ww-testdata/ww-testdata.wdl" as ww_testdata
+
+
+#### WORKFLOW DEFINITION ####
+
+workflow tritonnp_example {
+  meta {
+    author: "Chris Lo"
+    email: "clo2@fredhutch.org"
+    description: "TritonNP module."
+    url: "https://github.com/caalo/TritonNP"
+    outputs: {
+        final_composite: "Aggregrated outputs from all files",
+        fm_files: "Array of outputs from individal samples"
+    }
+  }
+
+  # Auto-download test data for testing purposes
+  call ww_testdata.download_tritonnp_data as download_demo_data { }
+
+  # Sample configuration
+  Array[String] sample_names = ["NA12878"]
+  Array[File] bam_paths = [download_demo_data.bam]
+  Array[File] bam_index_paths = [download_demo_data.bam_index]
+  Array[File] bias_paths = [download_demo_data.bias]
+  
+  # Global configuration parameters
+  String results_dir = "results"
+  File annotation = download_demo_data.annotation
+  File reference_genome = download_demo_data.reference
+  File reference_genome_index = download_demo_data.reference_index
+  File plot_list = download_demo_data.plot_list
+  Int map_quality = 20
+  String size_range = "15 500"
+  Int triton_main_ncpus = 4
+
+  # Process each sample
+  scatter (i in range(length(sample_names))) {
+    call triton_main {
+      input:
+        sample_name = sample_names[i],
+        bam_path = bam_paths[i],
+        bam_index_path = bam_index_paths[i],
+        bias_path = bias_paths[i],
+        annotation = annotation,
+        reference_genome = reference_genome,
+        reference_genome_index = reference_genome_index,
+        results_dir = results_dir,
+        map_quality = map_quality,
+        size_range = size_range,
+        cpus = triton_main_ncpus,
+        plot_list = plot_list
+    }
+  }
+
+  # Combine all feature matrices after all samples are processed
+  call combine_fms {
+    input:
+      fm_files = triton_main.fm_file,
+      results_dir = results_dir
+  }
+
+  output {
+    File final_composite = combine_fms.final
+    Array[File] feature_files = triton_main.fm_file
+  }
+}
+
+#### TASK DEFINITIONS ####
+
+# Task definitions file (triton_tasks.wdl)
+task triton_main {
+  meta {
+    description: "Task for running TritonNP"
+    outputs: {
+        fm_files: "Array of outputs from TritonNP"
+    }
+  }
+  parameter_meta {
+    sample_name: "Sample name"
+    bam_path: "Path to BAM file"
+    bam_index_path: "Path to BAM index file"
+    bias_path: "Path to GC corrected file from Griffin"
+    annotation: "Path to BED file of genomic region to process on"
+    reference_genome: "Path to Reference genome file"
+    reference_genome_index: "Path to Reference genome file index"
```

Comment:
```suggestion
    bam_path: "BAM file"
    bam_index_path: "BAM index file"
    bias_path: "GC corrected file from Griffin"
    annotation: "BED file of genomic region to process on"
    reference_genome: "Reference genome file"
    reference_genome_index: "Reference genome file index"
```
These are all type `File` so I would keep it at that to avoid confusion. Because by the time this task is running all these files have been localized. So whatever paths the user provided originally to the workflow are no longer accessible or meaningful to the task.